arduino-ide.sh

```diff
@@ -2,9 +2,12 @@
 
 FLAGS=''
 
-if [[ $XDG_SESSION_TYPE == "wayland" ]] && [ -c /dev/nvidia0 ]
-then
-    FLAGS="$FLAGS --disable-gpu-sandbox"
+if [[ $XDG_SESSION_TYPE == "wayland" ]]; then
+    if [ -c /dev/nvidia0 ]; then
```

Comment:
Has this been tested on a device with nvidia?
If i am reading this correctly it looks like wayland is only enabled if the device does not have an nvidia graphics card. Has nvidia + wayland been tested?
I can test it but it will take me some time to do so.

Reply:
Just tested this on nvidia and this needs modifying.
`[31 zypak-helper] Using spawn strategy test 1 as set by environment
Starting backend process. PID: 31
[3:0930/145117.287461:ERROR:browser_main_loop.cc(1400)] Unable to open X display.
/app/bin/arduino-ide: line 15:     3 Segmentation fault      (core dumped) env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/arduino-ide/arduino-ide /app/arduino-ide/resources/app/scripts/arduino-ide-electron-main.js $FLAGS "$@"`

Reply:
> Just tested this on nvidia and this needs modifying.
> 
> ```
> Starting backend process. PID: 31
> [3:0930/145117.287461:ERROR:browser_main_loop.cc(1400)] Unable to open X display.
> /app/bin/arduino-ide: line 15:     3 Segmentation fault      (core dumped) env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/arduino-ide/arduino-ide /app/arduino-ide/resources/app/scripts/arduino-ide-electron-main.js $FLAGS "$@"```
> ```

im going to double check this error isnt me but if possible could someone else test aswell.

Reply:
I think this can be fixed by changing the elif on line 8 to if.
(i have made that change locally and it appears to have fixed it. I am going to test on another computer with nvidia and if someone else is able to test that would be great!)